src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/UseValidPlatformStringTest.cs

```diff
@@ -0,0 +1,389 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.InteropServices.UseValidPlatformString,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.InteropServices.UseValidPlatformString,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+
+namespace Microsoft.NetCore.Analyzers.InteropServices.UnitTests
+{
+    public class UseValidPlatformStringTest
+    {
+        private const string s_msBuildPlatforms = "build_property._SupportedPlatformList=CustomPlatform";
```

Comment:
AFAIK, some editorconfig options doesn't do anything without something like `[*.cs]`.

@mavasani Can you confirm whether this is correct for this *specific* option?

If so, then there is a test gap.

Reply:
> AFAIK, some editorconfig options doesn't do anything without something like [*.cs].

hm, not sure, i was using it like this previously and it still works here too